src/components/AnnotationList/AnnotationList.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react';
+
+import Internationalize from '../Internationalize';
+import Annotation from '../Annotation';
+
+import './AnnotationList.scss';
+
+type Props = {
+    annotations: Annotations,
+    language?: string,
+    messages?: StringMap,
+    onDelete: Function
+};
+
+const AnnotationList = ({ annotations, language, messages: intlMessages, onDelete }: Props) => (
```

Comment:
Any reason you renamed `messages` here? Looks like it may be unnecessary.

Reply:
It's not necessary to rename it but usually messages refers to the `messages.js` w/ the localized objects for each translated string, intlMessage is more referring to the translated strings passed through the Internationalize component.